tests/facts/windows_logon_session_info/windows_logon_session_info.json

```diff
@@ -0,0 +1,406 @@
+{
+    "command": "Get-CimInstance -ClassName Win32_LogonSession | Format-List -Property *",
+    "output": [
+        "",
```

Comment:
Can we also trim this one down please? Ideally would keep fact tests under 100 lines of JSON unless absolutely necessary.

Reply:
Ok